index.ts

```diff
@@ -0,0 +1,260 @@
+import { AwesomeCordovaNativePlugin } from '@awesome-cordova-plugins/core';
```

Comment:
why is this new file needed?

Reply:
With this file we create the ngx file through awesome cordova plugin